packages/driver/src/cy/commands/actions/select.ts

```diff
@@ -11,6 +11,10 @@ const newLineRe = /\n/g
 export default (Commands, Cypress, cy) => {
   Commands.addAll({ prevSubject: 'element' }, {
     select (subject, valueOrTextOrIndex, options = {}) {
+      if (!_.isNumber(valueOrTextOrIndex) && !_.isString(valueOrTextOrIndex) && !_.isArray(valueOrTextOrIndex)) {
+        $errUtils.throwErrByPath('select.no_argument')
+      }
+
```

Comment:
If we detect the value is an array, we should look at its values and validate them as strings or numbers as well. It probably needs a separate error message, since arrays can't be nested inside.